elearning-backend/database/migrations/2022_08_01_120897_update_words_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateWordsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('words_learned', function(Blueprint $table) {
+            $table->dropForeign('words_learned_word_id_foreign');
+            $table->dropColumn('word_id');
+            $table->unsignedBigInteger('category_id');
+            $table->foreign('category_id')->references('id')->on('categories')->cascadeOnDelete();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('words_learned', function(Blueprint $table) {
+            $table->dropForeign('words_learned_word_id_foreign');
```

Comment:
```suggestion
            $table->dropForeign('category_id');
```
it should be category_id